piper/buttondialog.py

```diff
@@ -226,6 +240,9 @@ def _on_row_activated(self, listbox, row):
                 print("Unable to grab keyboard, can't set keystroke", file=sys.stderr)
             else:
                 self.stack.set_visible_child_name("capture")
+        elif row is self.row_left_hand_mode:
+            self._action_type = RatbagdButton.ACTION_TYPE_BUTTON
+            self._mapping = -2
```

Comment:
@whot (or any other reviewer): this part in particular I am unhappy with. There needs to be some kind of value to signal to the `ButtonsPage` that we want to swap buttons, and so for now I've decided on -2 because I can envision -1 being used by libratbag at some point, but every value is quite arbitrary. I'm not sure if there is any other, better approach that we can use without having to redo this whole dialog.